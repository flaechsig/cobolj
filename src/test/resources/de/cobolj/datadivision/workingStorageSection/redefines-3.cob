IDENTIFICATION DIVISION.
PROGRAM-ID. PERFTIMI.
ENVIRONMENT DIVISION.
DATA DIVISION.
WORKING-STORAGE SECTION.
  01 WS-NAME.
     05 WS-FST-NAME PIC X(5) VALUE 'HANS'.
     05 WS-SEC-NAME PIC X(10) VALUE 'FLAECHSIG'.
  01 WS-TOTAL-NAME REDEFINES WS-NAME PIC X(15).
  
PROCEDURE DIVISION.
	DISPLAY 'ACTUAL DATA IN WS-NAME : ' WS-NAME.
	DISPLAY 'ACTUAL DATA IN WS-TOTAL-NAME: ' WS-TOTAL-NAME.
	
	STOP RUN.