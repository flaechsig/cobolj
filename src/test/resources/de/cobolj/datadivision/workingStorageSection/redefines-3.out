ACTUAL DATA IN WS-NAME : HANS FLAECHSIG 
ACTUAL DATA IN WS-TOTAL-NAME: HANS FLAECHSIG 
